### ISSUE TITLE
Todo/Tasks Integration

### ISSUE THREAD

**Issue by Sparsa** [marcoancona/MineTime#148]
Hi, this is just a feature request. I felt the need for a todo/tasks option in this great application which will integrate with google tasks.

**Comment by zampanoo** [comment]
Seconded.

**Comment by funkypitt** [comment]
I agree, with subtasks support and proper integration with caldav (using "related to" for subtasks on caldav the way tasks.org does it on android).
